### PR TITLE
annotations: render all versions as one csv separated thing

### PR DESCRIPTION
**Pull request by tych0**

### CODE REVIEW

File: pod/annotations.go [hunk 1]
```diff
@@ -746,3 +746,17 @@ func IsScheduledInTrough(pod *corev1.Pod) bool {
 	_, ok := pod.Annotations[AnnotationKeyPodScheduledInTrough]
 	return ok
 }
+
+func RenderVersionInformation(kernelVersion string, executorVersion string) string {
```
Comment (solarkennedy) [inline pod/annotations.go:750]: What codebase were you imagining calling this function?

File: pod/annotations.go [hunk 2]
```diff
@@ -236,10 +236,10 @@ const (
...
 	// version recording; this is output from titus-executor mostly used
-	// for debugging. if we decide to add more versions here, let's make it
-	// one annotation with csv or similar.
-	AnnotationKeyRuntimeVersionTitusExecutor = "runtime.version.titus.netflix.com/titus-executor"
-	AnnotationKeyRuntimeVersionLinuxKernel   = "runtime.version.titus.netflix.com/linux-kernel"
+	// for debugging.
+	AnnotationKeyRuntimeVersions                = "runtime.titus.netflix.com/versions"
+	AnnotationMemberRuntimeVersionTitusExecutor = "titus-executor"
+	AnnotationMemberRuntimeVersionLinuxKernel   = "linux-kernel"
 )
```
Comment (solarkennedy) [inline pod/annotations.go:243]: I'm telling you, this is overkill.

Just have one annotation `AnnotationKeyRuntimeVersions` and let it be a free-form CSV until you know you need something more structured

File: pod/annotations.go (hunk 1 above)
Comment (tych0) [reply pod/annotations.go:750]: titus-executor has to do this, I can just move it to there.

File: pod/annotations.go (hunk 2 above)
Comment (tych0) [reply pod/annotations.go:243]: Sure, I can drop these.

**Review by tbak — APPROVED** [review]

**Pull request merged**